css-four-column-layout/styles.css

```diff
@@ -0,0 +1,57 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  background-color: rgb(247 248 250);
+  margin-left: -40px;
+  margin-right: -40px;
+  padding-left: 100px;
+  padding-right: 100px;
+}
+
+body {
+  font-family: Barlow, sans-serif;
+  text-align: center;
+}
+
+.column-quarter {
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+.column-full {
+  flex-basis: 100%;
+  text-align: center;
+}
```

Comment:
`column` classes should have `flex-basis` applied to them, and nothing else.

Reply:
![image](https://user-images.githubusercontent.com/117887912/208332765-1ccdd694-f118-451f-8a1f-e770a252d3f7.png)